### PR TITLE
updated Deneyap boards pin_arduino.h files

### PR DESCRIPTION
**Pull request by zeynepdicle**
*By completing this PR sufficiently, you help us to review this Pull Request quicker and also help improve the quality of Release Notes*

### Checklist
1. [ ] Please provide specific title of the PR describing the change, including the component name (*eg. „Update of Documentation link on Readme.md“*)
2. [ ] Please provide related links (*eg. Issue which will be closed by this Pull Request*)
3. [ ] Please **update relevant Documentation** if applicable
4. [ ] Please check [Contributing guide](https://docs.espressif.com/projects/arduino-esp32/en/latest/contributing.html)

*This entire section above can be deleted if all items are checked.*

-----------
## Description of Change
Please describe your proposed Pull Request and it's impact.

## Tests scenarios
Please describe on what Hardware and Software combinations you have tested this Pull Request and how.

(*eg. I have tested my Pull Request on Arduino-esp32 core v2.0.2 with ESP32 and ESP32-S2 Board with this scenario*)

## Related links
Please provide links to related issue, PRs etc.

(*eg. Closes #number of issue*)

### CODE REVIEW

**Review by P-R-O-C-H-Y — CHANGES_REQUESTED** [review]
Please just fix missing new line at the end of the file in [variants/deneyapmini/pins_arduino.h](https://github.com/espressif/arduino-esp32/pull/7338/files#diff-9de5913e149ceaa63e374882a16581519c92bc5a6661514312a38fc372326603) and [variants/deneyapkartg/pins_arduino.h](https://github.com/espressif/arduino-esp32/pull/7338/files#diff-5634121c78fad9825ddfe89d58fc6f2c4891c3816975ceb3fb1d28aa51934767).
After that will be ready to be merged :)

**Comment by zeynepdicle** [comment]
> Please just fix missing new line at the end of the file in [variants/deneyapmini/pins_arduino.h](https://github.com/espressif/arduino-esp32/pull/7338/files#diff-9de5913e149ceaa63e374882a16581519c92bc5a6661514312a38fc372326603) and [variants/deneyapkartg/pins_arduino.h](https://github.com/espressif/arduino-esp32/pull/7338/files#diff-5634121c78fad9825ddfe89d58fc6f2c4891c3816975ceb3fb1d28aa51934767). After that will be ready to be merged :)

fixed

**Comment by P-R-O-C-H-Y** [comment]
> 

@zeynepdicle Now there are missing new lines in more files.

**Comment by zeynepdicle** [comment]
> @zeynepdicle Now the new line at the end of file is missing in all pins_arduino.h files you changed. Please fix it.

Sorry @P-R-O-C-H-Y I don't understand. Which lines is missing ?

File: variants/deneyapkart/pins_arduino.h [hunk 1]
```diff
@@ -102,4 +101,11 @@ static const uint8_t MICC = 13;
...
 
-#endif /* Pins_Arduino_h */
+#define BUILTIN_LED LEDB
+#define LED_BUILTIN BUILTIN_LED // backward compatibility
+//#define RGB_BUILTIN LED_BUILTIN
+
+#define KEY_BUILTIN GPKEY
+#define BUILTIN_KEY KEY_BUILTIN
+
+#endif /* Pins_Arduino_h */
```
Comment (P-R-O-C-H-Y) [inline variants/deneyapkart/pins_arduino.h:111]: No newline at the end of file.

File: variants/deneyapkart1A/pins_arduino.h [hunk 2]
```diff
@@ -99,4 +95,13 @@ static const uint8_t SDMO = 14;
...
+static const uint8_t RGBLED  = 13;
+static const uint8_t GPKEY  = 0;
+
+#define RGB_BUILTIN RGBLED
+#define RGB_BRIGHTNESS 64
+
+#define KEY_BUILTIN GPKEY
+#define BUILTIN_KEY KEY_BUILTIN
+
+#endif /* Pins_Arduino_h */
```
Comment (P-R-O-C-H-Y) [inline variants/deneyapkart1A/pins_arduino.h:107]: No newline at the end of file.

File: variants/deneyapkartg/pins_arduino.h [hunk 3]
```diff
@@ -39,4 +37,16 @@ static const uint8_t D4 = 8;
...
+static const uint8_t RGBLED  = 10;
+static const uint8_t GPKEY  = 9;
+
+#define RGB_BUILTIN RGBLED
+#define RGB_BRIGHTNESS 64
+
+#define KEY_BUILTIN GPKEY
+#define BUILTIN_KEY KEY_BUILTIN
+
 #endif /* Pins_Arduino_h */
```
Comment (P-R-O-C-H-Y) [inline variants/deneyapkartg/pins_arduino.h:52]: No newline at the end of file.

File: variants/deneyapmini/pins_arduino.h [hunk 4]
```diff
@@ -24,43 +23,50 @@ static const uint8_t MOSI  = 40;
...
+static const uint8_t GPKEY	= 0;
+
+#define KEY_BUILTIN GPKEY
+#define BUILTIN_KEY KEY_BUILTIN
+
+#define BUILTIN_LED LEDB
+#define LED_BUILTIN BUILTIN_LED // backward compatibility
+//#define RGB_BUILTIN LED_BUILTIN
+
 #endif /* Pins_Arduino_h */
```
Comment (P-R-O-C-H-Y) [inline variants/deneyapmini/pins_arduino.h:72]: No newline at the end of file.

**Review by P-R-O-C-H-Y — COMMENTED** [review]
Add new line at the end of each pins_arduino.h file you edited.

**Comment by zeynepdicle** [comment]
is that true ?  @P-R-O-C-H-Y

**Review by P-R-O-C-H-Y — APPROVED** [review]

**Comment by P-R-O-C-H-Y** [comment]
> is that true ? @P-R-O-C-H-Y

Thanks for fixing, will merge asap as CI passes again :)

**Pull request merged**